build/NetFX.props

```diff
@@ -0,0 +1,6 @@
+﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net461' and '$(OS)' == 'Unix' ">
```

Comment:
We should use the MSBuild property function `[MSBuild]::IsOSPlatform` instead of this machinery since we are using MSBuild 15.3 (the version in the 2.0.0 SDK)